### ISSUE TITLE
Text cell editor doesn't respect the 'Tab Moves Focus' setting

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#14404]
Steps to Reproduce:

1. Open a notebook and create a text cell
2. Run the 'Toggle Tab key Moves Focus ' command
![image](https://user-images.githubusercontent.com/13777222/108914534-54c12400-75e0-11eb-8a08-a94f0bb61b7d.png)
3. Set the focus in the text editor and then press tab

Expected:
Just like in other editors, I should be able to move the keyboard focus out from the text cell

Actual:
focus stays in the text cell

**Comment by chlafreniere** [comment]
@alanrenmsft which edit mode are you in? WYSIWYG, or split/markdown view for the text cell?

**Comment by alanrenmsft** [comment]
I tried all 3 modes, behaves the same

**Comment by chlafreniere** [comment]
Whoa that's wild. Code and text cells use the same editor control. Good catch.
